raiden/tests/unit/transfer/mediated_transfer/test_mediatorstate.py

```diff
@@ -2135,79 +1645,46 @@ def test_mediator_lock_expired_after_receive_secret_reveal():
       and secrethashes_to_unlocklocks.
     """
-    amount = 10
-    block_number = 5
-    target = HOP2
     expiration = 30
     pseudo_random_generator = random.Random()
 
-    payer_channel = factories.make_channel(
-        partner_balance=amount,
-        partner_address=UNIT_TRANSFER_SENDER,
-        token_address=UNIT_TOKEN_ADDRESS,
-    )
-
-    payer_transfer = factories.make_signed_transfer_for(
-        payer_channel,
-        amount,
-        UNIT_TRANSFER_SENDER,
-        target,
-        expiration,
-        UNIT_SECRET,
-    )
-
-    payee_channel = factories.make_channel(
-        our_balance=amount,
-        token_address=UNIT_TOKEN_ADDRESS,
-        partner_address=target,
-    )
-    channel_map = {
-        payee_channel.identifier: payee_channel,
-        payer_channel.identifier: payer_channel,
-    }
-    possible_routes = [factories.route_from_channel(payee_channel)]
-
-    init_mediator = ActionInitMediator(
-        possible_routes,
-        factories.route_from_channel(payer_channel),
-        payer_transfer,
-    )
+    channels = mediator_make_channel_pair()
+    transfer = factories.make_default_signed_transfer_for(channels[0], expiration=30)
 
     iteration = mediator.state_transition(
-        None,
-        init_mediator,
-        channel_map,
-        pseudo_random_generator,
-        block_number,
+        mediator_state=None,
+        state_change=mediator_make_init_action(channels, transfer),
+        channelidentifiers_to_channels=channels.channel_map,
+        pseudo_random_generator=pseudo_random_generator,
+        block_number=5,
     )
 
-    transfer = payer_transfer
     secrethash = transfer.lock.secrethash
 
     block_lock_expired = expiration + DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS
 
-    assert secrethash in payer_channel.partner_state.secrethashes_to_lockedlocks
+    assert secrethash in channels[0].partner_state.secrethashes_to_lockedlocks
 
     # Reveal secret just before the lock expires
-    secret_reveal = ReceiveSecretReveal(UNIT_SECRET, target)
+    secret_reveal = ReceiveSecretReveal(UNIT_SECRET, UNIT_TRANSFER_TARGET)
 
     iteration = mediator.state_transition(
-        iteration.new_state,
-        secret_reveal,
-        channel_map,
-        pseudo_random_generator,
-        block_lock_expired,
+        mediator_state=iteration.new_state,
+        state_change=secret_reveal,
+        channelidentifiers_to_channels=channels.channel_map,
+        pseudo_random_generator=pseudo_random_generator,
+        block_number=block_lock_expired,
     )
 
     # Mediator should NOT send balance proof
     assert must_contain_entry(iteration.events, SendBalanceProof, {}) is None
 
     # Make sure the lock was moved
-    assert secrethash not in payer_channel.partner_state.secrethashes_to_lockedlocks
-    assert secrethash in payer_channel.partner_state.secrethashes_to_unlockedlocks
+    assert secrethash not in channels[0].partner_state.secrethashes_to_lockedlocks
```

Comment:
Maybe do the following before this line:
```
payer_channel, payee_channel = channels[0], channels[1]
```
This naming convention makes the test much easier to read than figuring out which is which